### PR TITLE
Prepare for 7.2.2 submission

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **chore**

### Description

- Prepare for 7.2.2 submission
- I created beta by following the beta freeze docs, the last beta we tested was last Friday "ios-7.2.2-2022.02.11.15" 

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

 #nochangelog

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ✅ **No changelog changes**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 15b1ced99ff20a0c09fb9d8f6e4b46d9eb0cee42
</p>

**Review by araujobarret — APPROVED** [review]
🙌

File: src/lib/Scenes/Artwork/Components/CommercialButtons/CommercialButtons.tests.tsx [hunk 1]
```diff
@@ -303,4 +303,19 @@ describe("CommercialButtons", () => {
     })
     expect(commercialButtons.find(Button).length).toEqual(0)
   })
+
+  it("renders both Make Offer and Contact Gallery buttons when isOfferable and isInquiriable", async () => {
```
Comment (brainbicycle) [inline src/lib/Scenes/Artwork/Components/CommercialButtons/CommercialButtons.tests.tsx:307]:
```suggestion
  it("renders both Make Offer and Contact Gallery buttons when isOfferable and isInquireable are true", async () => {
```
minor typo here

**Review by brainbicycle — APPROVED** [review]
tiny typo, looks good otherwise! thanks!

**Comment by MounirDhahri** [comment]
oups sorry, merged before I saw your comment @brainbicycle

**Pull request merged**